src/main/java/frc/robot/RobotContainer.java

```diff
@@ -276,7 +279,14 @@ public void populateSysIdDashboard() {
     tab.add("Tur Roll Quasi Rev", turretSubsystem.sysIdRollerQuasistaticCommand(kReverse)).withPosition(columnIndex, 1);
     tab.add("Tur Roll Dynam Fwd", turretSubsystem.sysIdRollerDynamicCommand(kForward)).withPosition(columnIndex, 2);
     tab.add("Tur Roll Dynam Rev", turretSubsystem.sysIdRollerDynamicCommand(kReverse)).withPosition(columnIndex, 3);
-
+  
+    // Testing
+    tab = Shuffleboard.getTab("Testing");
+    
+    tab.addNumber("Number of Tests Run", () -> testCommand.getTestState());
+    tab.add("Start LED Progress Bar", new LEDProgressBarCommand(ledSubsystem, testCommand::getTestState)).withPosition(0, 0);
+    tab.add("Start Testing", new TestCommand(intakeSubsystem, 
```

Comment:
You don't need two instances of `TestCommand`. Your LEDs aren't lighting up because `LEDProgressBarCommand` is calling a different instance of `TestCommand` than the one on ShuffleBoard. Just use `testCommand` here instead of a new instance and the LEDs will light up.